### PR TITLE
Enable checks for first_seen and daily tables

### PR DESCRIPTION
**Pull request by fbertsch**
Neither of these tables failed when we had the checks enabled previously.

### CODE REVIEW

File: dags/copy_deduplicate.py [hunk 1]
```diff
@@ -380,6 +381,7 @@
             "backfill",
             "*.baseline_clients_first_seen_v1",
             "--no-partition",
+            "--check",
```
Comment (fbertsch) [inline dags/copy_deduplicate.py:384]:
```suggestion
            "--checks",
```

File: dags/copy_deduplicate.py [hunk 2]
```diff
@@ -393,6 +395,7 @@
             "query",
             "backfill",
             "*.baseline_clients_daily_v1",
+            "--check",
```
Comment (fbertsch) [inline dags/copy_deduplicate.py:398]:
```suggestion
            "--checks",
```

File: dags/copy_deduplicate.py [hunk 3]
```diff
@@ -380,6 +381,7 @@
             "backfill",
             "*.baseline_clients_first_seen_v1",
             "--no-partition",
+            "--checks",
```
Comment (fbertsch) [inline dags/copy_deduplicate.py:384]: FYI @Iinh these were added here: https://github.com/mozilla/bigquery-etl/pull/4455/files

**Review by Iinh — APPROVED** [review]
LGTM, thanks!

**Pull request merged**